src/components/Basics/Hero/index.module.css

```diff
@@ -28,7 +28,7 @@
   bottom: -680px;
   right: -750px;
   width: 1500px;
-  z-index: -1;
+  @apply -z-1;
```

Comment:
Does this work cross-browser? https://caniuse.com/sr_css-apply

Reply:
this is a [tailwind directive](https://tailwindcss.com/docs/functions-and-directives#apply)